openvdb/Platform.h

```diff
@@ -205,8 +205,7 @@
 #if defined __GNUC__
```

Comment:
Could you please also change this line (205) to the following?
```
#if defined __INTEL_COMPILER
    #define OPENVDB_NO_TYPE_CONVERSION_WARNING_BEGIN
    #define OPENVDB_NO_TYPE_CONVERSION_WARNING_END
#elif defined __GNUC__
```
ICC actually defines `__GNUC__`, but it doesn't recognize GCC pragmas.

Reply:
Sure, done